internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -115,6 +115,14 @@ func (s *CreateRuntimeResourceStep) Run(operation internal.Operation, log logrus
 			}
 		}
 		log.Infof("Runtime resource %s/%s creation process finished successfully", operation.KymaResourceNamespace, runtimeResourceName)
+
+		newOp, backoff, _ := s.operationManager.UpdateOperation(operation, func(op *internal.Operation) {
+			op.Region = runtimeCR.Spec.Shoot.Region
```

Comment:
the region must be saved in the operation because next steps are using this field to set a label on Kyma resource